logstash/examples/default/values.yaml

```diff
@@ -0,0 +1,14 @@
+logstashConfig:
```

Comment:
This is probably in here just for your testing purposes. However it would be nice to keep the "default" example to only use the default values (so no values). This example mostly exists to make sure that the default install results in a working and healthy deployment. For logstash its a bit weird since it won't be doing anything by default. However it is still nice to have a test making sure that `helm install --name logstash elastic/logstash` doesn't result in logstash failing to startup.

Reply:
Fixed in https://github.com/elastic/helm-charts/pull/333/commits/e2f43469596af1fa50788eaff236f38f29318a1c